Tests/SwiftDocCTests/Test Bundles/AvailabilityBundle.docc/ComplexAvailableDeprecated.md

```diff
@@ -0,0 +1,13 @@
+# Complex Available Deprecated Article
```

Comment:
`ComplexAvailableDeprecated.md` I guess can be replaced with the already existing tests in `testValidDirective`  since it looks like this:

```swift
        let source = """
         @Metadata {
             @Available(macOS, introduced: "11.0")
             @Available(iOS, introduced: "15.0")
             @Available(watchOS, introduced: "7.0", deprecated: "9.0")
             @Available(*, introduced: "0.1", deprecated: "1.0")
         }
         """
         try assertValidMetadata(source: source)
 ```